test/debts.js

```diff
@@ -32,6 +41,15 @@ contract('Debts', function(accounts) {
     return Promise.resolve()
     .then(() => debts.borrow(value, {from: borrower}))
     .then(() => asserts.throws(debts.borrow(1, {from: borrower})));
+  }); 
+
+   it('added: should fail on underflow when repaing', () => {
+    const borrower = accounts[3];
+    const value = 100;
+    const underValue = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
```

Comment:
In order for underflow to happen, you just need a number bigger than value itself, say 101. But this also works.